node/src/main/java/eu/cloudnetservice/node/service/defaults/AbstractService.java

```diff
@@ -782,6 +785,33 @@ protected void downloadInclusionFile(@NonNull ServiceRemoteInclusion inclusion,
       this.serviceId().nodeUniqueId()};
   }
 
+  protected void initStandardServiceLogHandler() {
+    this.logCache.addHandler((_, line, stderr) -> {
+      for (var logTarget : this.logTargets) {
+        if (logTarget.first().equals(ChannelMessageSender.self().toTarget())) {
```

Comment:
I know that this was just moved, but can we create the ChannelMessage target once instead of each time?